core/src/main/java/org/apache/accumulo/core/client/impl/TabletLocator.java

```diff
@@ -96,19 +96,19 @@ public static synchronized void clearLocators() {
     locators.clear();
   }
 
-  public static synchronized TabletLocator getLocator(ClientContext context, Text tableId) {
+  public static synchronized TabletLocator getLocator(ClientContext context, String tableId) {
```

Comment:
For external projects that are using this, I am happy 1.8 finally has locator in the public API.